sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/tests/Samples/Sample01_ReadmeSnippets.cs

```diff
@@ -11,54 +11,59 @@
 
 namespace Microsoft.Azure.Data.SchemaRegistry.ApacheAvro.Tests.Samples
 {
-    [Ignore("Only verifying that the sample builds")]
     public class Sample01_ReadmeSnippets : SamplesBase<SchemaRegistryClientTestEnvironment>
     {
-        [Ignore("Only verifying that the sample builds")]
-        [Test]
+#pragma warning disable IDE1006 // Naming Styles
+        private SchemaRegistryClient schemaRegistryClient;
+#pragma warning restore IDE1006 // Naming Styles
+        private byte[] _memoryStreamBytes;
```

Comment:
With the way that `_memoryStreamBytes` is used, it looks like you're relying on the tests running in a specific order and not being parallelized.    You may want to consider adding `NonParallelizable` and tagging the tests with `Order` to ensure deterministic execution.

Reply:
`NonParallelizable` is on the base sample class (part of the core test framework). I didn't know about `Order`. Instead, I just ran the tests within the other tests. I'll use `Order` instead.

Reply:
Added Order attributes.